tests/test_maintel_take_image_anycam.py

```diff
@@ -0,0 +1,403 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import contextlib
+import unittest
+from types import SimpleNamespace
+
+# import pytest
+from lsst.ts import standardscripts, utils
+from lsst.ts.standardscripts.maintel import CameraSetup, TakeImageAnyCam
+
+
+class TestTakeImageAnyCam(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    async def basic_make_script(self, index):
+        self.script = TakeImageAnyCam(index=index)
+
+        # Mock MTCS instances and its methods
+        self.script.mtcs = unittest.mock.AsyncMock()
+        self.script.mtcs.start_task = utils.make_done_future()
```

Comment:
You can remove this. It would only be called in the `configure` method if `mtcs` is not defined.